bundles/mapping/infobox/plugin/openlayerspopup/OpenlayersPopupPlugin.ol.js

```diff
@@ -526,13 +582,13 @@ Oskari.clazz.define(
                     // No content left, close popup
                     this.close(popupId);
                 } else {
-                    const { colourScheme, font, title, lonlat } = popup;
+                    const { title, lonlat, options } = popup;
                     this._renderPopup(
                         popupId,
                         contentData,
                         title,
                         lonlat,
-                        { colourScheme, font },
+                        options,
```

Comment:
Just to make sure we don't try to access the keys on undefined object:
```suggestion
                        options || {},
```